hawkbit-rest-core/src/test/java/org/eclipse/hawkbit/rest/util/JsonBuilder.java

```diff
@@ -33,6 +35,62 @@
  */
 public abstract class JsonBuilder {
 
+    public static String ids(final Collection<Long> ids) {
+        final JSONArray list = new JSONArray();
+        for (final Long smID : ids) {
+            list.put(new JSONObject().put("id", smID));
+        }
+
+        return list.toString();
+    }
+
+    public static String controllerIds(final Collection<String> ids) {
+        final JSONArray list = new JSONArray();
+        for (final String smID : ids) {
+            list.put(new JSONObject().put("controllerId", smID));
+        }
+
+        return list.toString();
+    }
+
+    public static String tags(final List<Tag> tags) throws JSONException {
+        final StringBuilder builder = new StringBuilder();
+
+        builder.append("[");
+        int i = 0;
+        for (final Tag tag : tags) {
+            try {
+                createTagLine(builder, tag);
+            } catch (final Exception e) {
+                e.printStackTrace();
```

Comment:
Well, the code is copied from the other builders but I actually don't know why the exception is watched. I will fix this in the entire class.